docker_start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Start script for docs.developer.ch.gov.uk
+
+PORT=8080
```

Comment:
The port is hard-coded here but this is assigned as a build ARG in the Dockerfile - should the build ARG be assigned to an ENV and this updated to use the ENV instead?

Reply:
ARG removed from dockerfile